package-system/AWSGameLiftServerSDK/build_package_image.py

```diff
@@ -0,0 +1,289 @@
+        shared_libs_extension: str = ".so"
+        static_libs_extension: str = ".a"
+    else:
+        raise Exception(f"Error unsupported platform: {PACKAGE_PLATFORM}")
+    
+    install_folder: pathlib.Path = build_path.joinpath("prefix")
+    
+    if lib_type == PACKAGE_LIB_TYPES[0]:
+        destination: pathlib.Path = libs_output_path.joinpath(f"bin/{build_type}")
+    else:
+        destination: pathlib.Path = libs_output_path.joinpath(f"lib/{build_type}")
+    destination.mkdir(parents=True)
+    
+    if lib_type == PACKAGE_LIB_TYPES[0]:
+        for file in install_folder.joinpath("bin").glob(f"*{shared_libs_extension}"):
+            shutil.copy2(file.resolve(), destination.resolve())
+    
+    for file in install_folder.joinpath("lib").glob(f"*{static_libs_extension}"):
+        shutil.copy2(file.resolve(), destination.resolve())
+
+def build_gamelift_server_sdk(working_directory: WorkingDirectoryInfo,
+                              build_type: str,
+                              lib_type: str) -> None:
+    build_folder: pathlib.Path = working_directory.build_path.joinpath(f"{build_type}_{lib_type}").resolve()
+    
+    print(f"Generating GameLift Server SDK project with {build_type} {lib_type} configuration...")
+    configure_sdk_project(working_directory.source_path.resolve(), build_folder.resolve(), build_type, lib_type)
+    
+    print(f"Building GameLift Server SDK project with {build_type} {lib_type} configuration...")
+    build_sdk_project(working_directory.source_path.resolve(), build_folder.resolve(), build_type)
+    
+    print(f"Copying {build_type} {lib_type} built sdk libs into output folder...")
+    copy_sdk_libs(working_directory.libs_output_path, build_folder, build_type, lib_type)
+
+def generate_packageInfo(working_directory: WorkingDirectoryInfo) -> None:
+    settings={
+        'PackageName': f'{PACKAGE_NAME}-{PACKAGE_VERSION}-{PACKAGE_PLATFORM}',
+        "URL"         : f'{PACKAGE_URL}',
+        "License"     : f'{PACKAGE_LICENSE}',
+        'LicenseFile': f'{PACKAGE_NAME}/{PACKAGE_LICENSE_FILE}'
+    }
+    package_file: str = working_directory.output_path.joinpath("PackageInfo.json").resolve()
+    with package_file.open('w') as fh:
+        json.dump(settings, fh, indent=4)
+
+def generate_cmake_file(working_directory: WorkingDirectoryInfo) -> None:
+    cmake_file_source: pathlib.Path = PACKAGE_BASE_PATH.joinpath(f"Find{PACKAGE_NAME}.cmake.{PACKAGE_PLATFORM.title()}")
+    if cmake_file_source.is_file():
+        find_cmake_content = cmake_file_source.read_text("UTF-8", "ignore")
+
+        target_cmake_file: pathlib.Path = working_directory.output_path.joinpath(f"Find{PACKAGE_NAME}.cmake")
+        target_cmake_file.write_text(find_cmake_content)
+    else:
+        raise Exception(f"Error finding cmake source file: {cmake_file_source.resolve()}")
+
+
+if __name__ == '__main__':
+    try:
+        print("Collecting package build info for GameLift Server SDK...")
```

Comment:
Prefer python logging over print

Reply:
will update

Reply:
going to keep print, tried logging, but it just overcomplicated the use case here.
The purpose is just print out to terminal, and logging requires extra step to setup handler for doing it.